bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -10,8 +10,7 @@ private func dummy() {
     _ = NSLocalizedString("%s block, newly available", comment: "translators: Newly available block name. %s: The localized block name")
     _ = NSLocalizedString("%s block. Empty", comment: "translators: accessibility text for the media block empty state. %s: media type")
     _ = NSLocalizedString("%s block. This block has invalid content", comment: "translators: accessibility text for blocks with invalid content. %d: localized block title")
-    _ = NSLocalizedString("%s converted to regular block", comment: "translators: %s: name of the reusable block")
-    _ = NSLocalizedString("%s converted to regular blocks", comment: "translators: %s: name of the reusable block")
+    _ = NSLocalizedString("%s detached", comment: "translators: %s: name of the synced block")
```

Comment:
All strings related to reusable blocks have been updated due to [this expected change](https://github.com/WordPress/gutenberg/pull/51704).